bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java

```diff
@@ -175,6 +175,26 @@ public void handleCommand(ChannelUID channelUID, Command receivedCommand) {
                             command = new DecimalType(((QuantityType<?>) command).toBigDecimal());
                         }
                     }
+                    if (paramType == CommandParameterType.OPENCLOSENUMBER) {
+                        if (command instanceof OpenClosedType) {
+                            value = new JsonPrimitive(command == OnOffType.ON ? 1 : 0);
```

Comment:
Looks weird. If command is instance of OpenClosedType, its value will never be OnOffType.ON.

Reply:
totally right! too much copy pasting:-(